ubuntudesign/www.ubuntu.com
# Remove redundant link to calculator

Fixes #1209.

Also remove a redundant sentence, and the old Calculator JavaScript.

## QA

Run the site, visit `/cloud/plans-and-pricing`, check the link to calculator is gone. Check the wording still makes sense to you.
